src/foraging_gui/Foraging.py

```diff
@@ -147,6 +153,31 @@ def __init__(self, parent=None,box_number=1,start_bonsai_ide=True):
             self._ReconnectBonsai()   
         logging.info('Start up complete')
     
+    def _load_rig_metadata(self):
+        '''Load the latest rig metadata'''
+        if not os.path.exists(self.rig_metadata_folder):
+            self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata found!')
+            return
+        json_files = [f for f in os.listdir(self.rig_metadata_folder) if f.endswith('.json')]
+        dates=[]
+        for file in json_files:
+            file_path = os.path.join(self.rig_metadata_folder, file)
+            #Assume the file name has the structure 'rig' + rig name+'_'+date+'.json'
+            if file.startswith('rig'+self.current_box):
+                date_str = file.split('_')[-1].split('.')[0]
+                dates.append(date_str)
+        if len(dates)==0:
+            logging.warning('No rig metadata files found')
+            self.latest_rig_metadata_file=''
+            self.latest_rig_metadata={}
+            self._manage_warning_labels(self.MetadataWarning,warning_text='No rig metadata found!')
```

Comment:
We should log an error if no rig metadata is found

Reply:
It was logged `logging.warning('No rig metadata file found')`

Reply:
Right, I mean that we should change the `warning` to an `error`

Reply:
Done